innodb-java-reader-cli/src/main/java/com/alibaba/innodb/java/reader/cli/CsvPrinter.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Alibaba Group Holding Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.innodb.java.reader.cli;
+
+import java.io.IOException;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.QuoteMode;
+
+/**
+ * CSV printer.
+ * 
+ * @author xu.zx
+ * @author Adam Jurcik
+ */
+public class CsvPrinter {
+
+  public static final CSVFormat FIELD_FORMAT_QUOTED;
+
+  static {
+    // delimiter is not used
+    FIELD_FORMAT_QUOTED = CSVFormat.newFormat(',')
```

Comment:
By default, what you set makes sense. But in mysql dump file, `null` is treated as `NULL`, would it be better to make that an option?
Also, for `QuoteMode.ALL_NON_NULL`, could you make it as an option in CLI?
I suggest resolve input to enum by the following method call:
```
// org.apache.commons.lang3.EnumUtils
QuoteMode quoteMode = EnumUtils.getEnum(QuoteMode.class, "NON_NUMERIC");
```

Reply:
Well, this started to be kind of a pain as the project did not support proper CSV printing before. For example:
* the project's code printed lower case `null`s for null values before,
* the `commons-csv` lib needs an escaping character when you choose not to escape the values, which is the project's default way.

I implemented the CLI options as you suggested. However, the escape character option may be missing now. Furthermore, the project supports delimiters as strings, i.e. not only characters, so that the `common-csv` lib cannot be used to properly quote such delimiters within field values.

To summarize, I am not happy about the additional changes. Instead, I would suggest to either (1) use a standard Linux tool such as `sed` to handle the issue with `null` vs. `NULL` or (2) cover the CSV printing in a much more complete way in the project. However, the (2) exceeds a lot the purpose for which I made the change, sorry.

Sorry for not updating the README. I am ready to do it.

Reply:
I learned your concern, for the null, I agree it it better to be processed by external tool, for the QuoteMode, it is ok to quote all non-null fields.